palindrom-client.html

```diff
@@ -535,13 +537,15 @@
         var n = ~~Number(str);
         return String(n) === str && n >= 0;
     }
+    /* IE and Safari, polyfill Object.assign */
+    var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
```

Comment:
Is that needed? We don't support IE. `Object.assign` is available in Safari 9. We support only current versions of Edge, Safari, Firefox and Chrome.